.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 build
 secrets.*.js
 *.log
+.idea/
```

Comment:
I'd prefer to have this in a global gitnore instead of in the project's gitignore https://github.com/crookedneighbor/shambleshark/pull/45/commits/877c72c7993b06ba6f9c451157ed288386c7404e